Assets/00_Content/Scripts/Interactables/Kitchens/Kitchen.cs

```diff
@@ -59,5 +67,14 @@ public class Kitchen : Interactable {
 
 			AudioManager.Instance.PlayEffect(SoundEffect.FoodReady);
 		}
+		/// <summary>
+		/// Resets the progress etc on the kitchen between each round
+		/// </summary>
+		private void HandleOnNewRoundStart() {
+			isCooking = false;
+			//Might actually not need to reset cookingProgress to zero here, seeing as that is done whenever StartCooking() begins.
+			cookingProgress = 0;
+			cookingProgressBar.Hide();
+		}
```

Comment:
This will probably need to be updated with the new changes to kitchen queueing. This can be done after resolving the merge conflicts.